Support policies performing revers DNS lookups

flavio opened the pull request:
Update to latest version of policy-evaluator, so that new waPC functions are supported.

This is needed to fix https://github.com/kubewarden/policy-server/issues/236

viccuad reviewed (approved):
LGTM, Straight bump (needs to consume correct dep in cargo.toml)

ereslibre reviewed (commented):
LGTM modulo the tiny "reverse" detail in the commit description. With that meaning: I think we should get rid of the "reverse" word given we are doing regular DNS resolutions.

flavio commented:
I've updated the `Cargo.toml` to reference the just released version of `policy-evaluator`. I'm going to merge it once all the tests are green.

BTW, I've also fixed the commit message as pointed out by @ereslibre

flavio commented:
GH got stuck during the e2e tests because of unknown reasons. All the tests are green locally, going ahead with the merge
